nifi-nar-bundles/nifi-asn1-bundle/nifi-asn1-services/src/main/java/org/apache/nifi/jasn1/JASN1Reader.java

```diff
@@ -366,13 +387,13 @@ public void reportError(RecognitionException e) {
             }
         };
 
+        AsnModel model = new AsnModel();
+        parser.module_definitions(model);
+
         if (parseError.get()) {
```

Comment:
The reportError can be - and should be - called multiple times. We want to know as much errors as possible. Throwing the exception there would allow to report only the first encountered issue.

Reply:
I see, makes sense!